cmd/kubectl-gadget/networkpolicyadvisor.go

```diff
@@ -53,30 +53,30 @@ var networkPolicyReportCmd = &cobra.Command{
 }
 
 var (
-	inputFileName  string
-	outputFileName string
-	namespaces     string
+	npaInputFileName  string
+	npaOutputFileName string
+	npaNamespaces     string
 )
 
 func init() {
 	rootCmd.AddCommand(networkPolicyCmd)
 
 	networkPolicyCmd.AddCommand(networkPolicyMonitorCmd)
-	networkPolicyMonitorCmd.PersistentFlags().StringVarP(&outputFileName, "output", "", "-", "File name output")
-	networkPolicyMonitorCmd.PersistentFlags().StringVarP(&namespaces, "namespaces", "", "", "Comma-separated list of namespaces to monitor")
+	networkPolicyMonitorCmd.PersistentFlags().StringVarP(&npaOutputFileName, "output", "", "-", "File name output")
+	networkPolicyMonitorCmd.PersistentFlags().StringVarP(&npaNamespaces, "npaNamespaces", "", "", "Comma-separated list of npaNamespaces to monitor")
```

Comment:
Too brutal `s///`?
Also, I am concerned this is a problem but I would like to first have #500 merged before doing it.
Particularly because I am sure we can group some variables per gadget type.

Reply:
> Too brutal s///?

ops, replaced too many things. 

> Also, I am concerned this is a problem but I would like to first have https://github.com/kinvolk/inspektor-gadget/pull/500 merged before doing it.
Particularly because I am sure we can group some variables per gadget type.

makes sense to me.